src/app/api/legacy/localRepoServiceHandlers.ts

```diff
@@ -8,7 +8,7 @@ import { READ_FILE, CONTROLLER_API_ENDPOINT, DataPlaneStatusCode, GET_DIRECTORIE
 import { ModelDefinitionNotFound } from '../models/modelDefinitionNotFoundError';
 import { ModelDefinitionNotValidJsonError } from '../models/modelDefinitionNotValidJsonError';
 
-export class LocalRepoServiceHandler implements DirectoryInterface, ModelRepositoryInterface {
+export class LocalRepoServiceHandlers implements DirectoryInterface, ModelRepositoryInterface {
```

Comment:
The plural here seems a little weird -- may give wrong impression.  We can always split the handlers into two types if you are concerned about the (s) with respect to DirectoryInterface / ModelRepositoryInterface